src/go/k8s/pkg/console/admin.go

```diff
@@ -130,7 +137,9 @@ func getTLSConfigOpt(cluster *redpandav1alpha1.Cluster, store *Store) (kgo.Opt,
 	return kgo.DialTLSConfig(config), nil
 }
 
-func clientCert(cluster *redpandav1alpha1.Cluster, store *Store) ([]tls.Certificate, error) {
+func clientCert(
```

Comment:
I wish we've done all these formatting changes in separate PR 😅 would make review easier

Reply:
Yeah, I'm not even sure where they came from. I think I used a newer linter and it did that automatically.